mysqld_exporter.go

```diff
@@ -696,6 +766,78 @@ func (e *Exporter) scrape(ch chan<- prometheus.Metric) {
 		}
 	}
 
+	if *perfEventsStatements {
+		perfQuery := fmt.Sprintf("%s LIMIT %s", perfSchemaEventsStatementsQuery, strconv.Itoa(*perfEventsStatementsLimit))
```

Comment:
Ah, I thought about adding a `%d` into the `perfSchemaEventsStatementsQuery` but this works just as well.
You can replace the second `%s` with `%d` and remove the call to `strconv.Itoa`.